.github/pull_request_template.md

```diff
@@ -29,18 +29,19 @@ Please also list any relevant details for your test configuration.
 
 **Test Configuration**:
 
-- sage versions:
-- os:
-- browsers:
-- screen readers:
-- misc:
+- Sage versions:
+- OS:
+- Browsers:
+- Screen readers:
+- Misc:
 
 # Checklist:
 
 - [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have performed a self-review of my code
+- [ ] I have commented on my code, particularly in hard-to-understand areas
```

Comment:
I believe it was better the other way. I comment my code, as I don't comment on my code.

Reply:
I will blame Grammarly for yelling at me for this one but secondarily then blame myself for not catching it. 
In any event, I agree, changing it back.